project-management-application-web/src/app/app.module.ts

```diff
@@ -9,13 +9,15 @@ import {ErrorPageComponent} from "./error-page/error-page.component";
 import {PageNotFoundComponent} from "./page-not-found/page-not-found.component";
 import {HTTP_INTERCEPTORS, HttpClientModule} from "@angular/common/http";
 import {XhrInterceptor} from "./xhr.interceptor";
+import {HeaderComponent} from "./header/header.component";
 
 @NgModule({
   declarations: [
     AppComponent,
     LoginComponent,
     ErrorPageComponent,
-    PageNotFoundComponent
+    PageNotFoundComponent,
+    HeaderComponent,
```

Comment:
you should use modules, not components. this wont scale, adding more and more components